avm-transpiler/src/transpile.rs

```diff
@@ -252,8 +252,11 @@ pub fn brillig_to_avm(brillig: &Brillig) -> Vec<u8> {
                     ],
                     ..Default::default()
                 });
+            },
+            BrilligOpcode::ForeignCall { function, destinations, inputs } => {
+                handle_foreign_call(&mut avm_instrs, function, destinations, inputs);
             }
-            _ => panic!(
+        _ => panic!(
```

Comment:
Don't think we want this spacing change. Should be same indentation as other cases above?